packages/base-styles/_mixins.scss

```diff
@@ -154,26 +146,17 @@
 	outline-offset: 2px;
 }
 
-// Formatting Buttons.
-@mixin formatting-button-style__hover {
-	color: $dark-gray-500;
-	box-shadow: inset 0 0 0 1px $dark-gray-500, inset 0 0 0 2px $white;
-}
-
-@mixin formatting-button-style__active() {
-	outline: none;
-	color: $white;
-	box-shadow: none;
-	background: $dark-gray-500;
-}
 
-@mixin formatting-button-style__focus() {
-	box-shadow: inset 0 0 0 1px $dark-gray-500, inset 0 0 0 2px $white;
+/**
+ * Block Toolbar/Formatting Buttons
+ */
 
-	// Windows High Contrast mode will show this outline, but not the box-shadow.
-	outline: 2px solid transparent;
+@mixin block-toolbar-button-style__focus() {
```

Comment:
I have the same concern raise about the previous mixin. Should we just duplicate in Popover?

Reply:
The mixin conversation I feel is worth elevating above this PR. They are intended to help create a single place to define reused code. But at the same time I can see how that goes against the CSS & components mantra.